src/components/forms/SignInForm.js

```diff
@@ -1,55 +1,82 @@
-import React, { Component } from 'react';
-import LocalApi from '../../apis/LocalApi'
+import React, { Component } from "react";
+import LocalApi from "../../apis/LocalApi";
+import { connect } from "react-redux";
+import { setAuthToken } from "../../actions";
 
 class SigninForm extends Component {
-//we create a class. 
-//we create an event handler. 
-    onFormSubmit = (event) => {
-//we prevent the default behaviour of our event, 
-        event.preventDefault();
-//we create a variable that contains the state of our username and password. 
-        const {username, password} = this.state
-//we use axios through our import of LocalApi, to post to our backend the username and password. 
-        LocalApi.post("/users/login", { username, password })
-//if we successfully post to our backend we then return to the home page. 
-       .then(response => {
-         // Redirect to login page for now.
-         console.log(response)
-         this.props.history.push("/");
-       })
-//otherwise, we console.log what our error is. 
-       .catch(error => {
-           console.log(error);
-           console.log("not working")
-           this.setState({ error });
-       });
+  state = {
+    username: "",
+    password: ""
+  };
+
+  onFormSubmit = async event => {
+    event.preventDefault();
+    const { username, password } = this.state;
+
+    // LocalApi.post("/users/login", { username, password })
+    //   .then(response => {
+    //     console.log(response);
+    //     this.props.setAuthToken(response.data);
+    //     this.props.history.push("/");
+    //   })
+    //   .catch(error => {
+    //     console.log(error);
+    //     console.log("not working");
+    //     this.setState({ error });
+    //   });
```

Comment:
I think it is safe to remove this code now.

Reply:
OK! Thanks :)